pyproject.toml

```diff
@@ -66,7 +66,7 @@ joblib = "*"
 plotly = "*"
 hydra_slayer = "*"
 matplotlib = "*"
-typing_extensions = "*"
+typing_extensions = "4.12.2"
```

Comment:
I don't agree with this change. We allow `typing_extensions` to have minimal version that doesn't lead to any error. It will be smth like
`typing_extensions = "^4.1.0"`

To update version in lock it will be enough to call `poetry update typing_extensions` or smth like this. It will install the latest available version according to the `pyproject.toml` that doesn't conflict with other packages.

Reply:
Sure this is off, it shouldn't be like this